packages/xstate-wallet/package.json

```diff
@@ -72,6 +72,7 @@
     "url-loader": "3.0.0",
     "url-parse": "1.4.7",
     "webpack": "4.41.5",
+    "webpack-dev-server": "^3.11.0",
```

Comment:
Surprised this wasn't here 😳

Reply:
I'm thinking we need an integration test in this repo to check for this stuff before we publish it. There are other things that are currently broken.